gax/src/main/java/com/google/api/gax/batching/v2/BatcherImpl.java

```diff
@@ -198,5 +223,10 @@ void onBatchFailure(Throwable throwable) {
         }
       }
     }
+
+    boolean hasAnyThresholdReached() {
+      return (elementThreshold != 0 && elementThreshold <= elementCounter)
+          || (bytesThreshold != 0 && bytesThreshold <= byteCounter);
```

Comment:
I want it to immediately flush on if threshold is zero

Reply:
I am sorry, I assumed disable meaning disabling the threshold(though that can be done by setting a very high value). 
Just a QQ: If the user sets any threshold to 0 then all other thresholds will be ignored. Is this what we want to add here... right?